dashboard/app/controllers/weblab_host_controller.rb

```diff
@@ -1,4 +1,5 @@
 class WeblabHostController < ApplicationController
+  layout false
```

Comment:
Nit: any reason to make this controller-wide instead of putting it inside the `def index` action?